.github/workflows/ci-pipeline.yml

```diff
@@ -379,21 +379,21 @@ jobs:
       - if: ${{ failure() }}
         uses: ./.github/actions/run-failure-log
 
-  Run_dpdk-21_11:
-    runs-on: ubuntu-22.04
+  Run_dpdk-22_11:
```

Comment:
The DEPENDENCIES file still says that DPDK versions 21.11 and 23.11 are supported, so I am not sure dropping them from CI (especially 21.11) is ok unless the minimum version mentioned in DEPENDENCIES is updated too. I guess version 23.11 is included in Ubuntu 24.04, so its gets a little bit of test exposure through Run_gcc and Run_clang.

Reply:
Looks like Ubuntu 24.04 container is still using DPDK v22.11.6 (should be updated to v23.11.x). The same test jobs should be included also for the arm64 pipeline.